scipy/integrate/meson.build

```diff
@@ -81,7 +81,7 @@ dop_lib = static_library('dop_lib',
     fortran_args: fortran_ignore_warnings)
 
 py3.extension_module('_quadpack',
-    ['__quadpack.h', '_quadpackmodule.c'] + quadpack_src + mach_src,
+    ['__quadpack.h', '_quadpackmodule.c'],
```

Comment:
Just for my instruction - these were unnecessary because the 'link_with' line further down specifies `quadpack_lib, mach_lib`, and these specify `quadpack_src` and `mach_src` in their dependencies?

Reply:
Yes exactly. And the reason that `mach`, `quadpack` et al. are built as separate static libraries first is because they're linked into several Python extensions, and we only want to build the source files once.